virttest/qemu_vm.py

```diff
@@ -2786,7 +2633,9 @@ def create(self, name=None, params=None, root_dir=None,
 
             # Create serial ports.
             for serial in params.objects("serials"):
-                self.serial_ports.append(serial)
+                if params.object_params(serial).get(
+                        "serial_type") != 'virtio-serial':
```

Comment:
We don't use `virtio-serial` now,  we use `virtserialport`, `virtconsole` and so on.